code/modules/power/smes.dm

```diff
@@ -312,10 +313,6 @@
 	return
 
 
-/obj/machinery/power/smes/add_load(amount)
```

Comment:
What does removing this line do? Make sure you thoroughly test the smes, take it apart, check the input and output wires, ensure the terminals disconnect properly, etc.

Reply:
Power already has a proc for add_load(amount). Since this one was only called once in smes, the parent one is now called on line 249 instead. It does the same thing, but removes a couple checks that already happen anyway, and hands adjusting the load var over to the object that owns it. SMES was thoroughly tested.